README.md

```
@@ -412,6 +434,115 @@ docker-compose down -v --rmi all
+### 📝 Variáveis de Ambiente Necessárias
+
+```bash
+# Obrigatória
+GEMINI_API_KEY=sua_chave_gemini_aqui
+
+# Opcionais (Supabase)
+SUPABASE_URL=https://seu-projeto.supabase.co
+SUPABASE_ANON_PUBLIC=sua_chave_publica
+```
+
+**Obter API Key**: https://aistudio.google.com/app/apikey
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)